exercises/concept/last-will/.docs/introduction.md

```diff
@@ -2,7 +2,7 @@
 
 ## Namespaces
 
-An important method for code organization are namespaces.
+An important method for code organization is namespaces.
```

Comment:
While grammatically correct, I think your proposed version sounds a git odd ("is" followed by plural). How about this:
```suggestion
An important method for code organization is the use of namespaces.
```